Gems/EMotionFX/Code/Source/Editor/SkeletonModel.cpp

```diff
@@ -389,7 +420,7 @@ namespace EMotionFX
             break;
         }
         case ROLE_NODE_INDEX:
-            return qulonglong(nodeIndex);
+            return isRootNode ? qulonglong(-1) : qulonglong(nodeIndex);
```

Comment:
`qulonglong(-1)` should probably be `EMotionFX::InvalidIndex` as that is the constant that we use for indicating invalid node/joint indices. `Node::m_nodeIndex` is also initialized with `EMotionFX::InvalidIndex` in Node.cpp for instance.